src/transformStateWithClones.js

```diff
@@ -7,7 +7,36 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let statedCopy = { ...state };
+  const ADDPROP = 'addProperties';
+  const REMOVEPROP = 'removeProperties';
+  const CLEAR = 'clear';
+  const ACTIONSRESULT = [];
```

Comment:
it is better to use UPPER_SNAKE_CASE when you use constants, also do not use contractions, example: 
ADDPROP -> ADD_PROPERTY